### ISSUE TITLE
How about use "bottom" instead of "height" for AdjustBottomMargin, Android?

### ISSUE THREAD

**Issue by WolfgangKurz** [gree/unity-webview#1075]
With `UNITYWEBVIEW_ANDROID_FORCE_MARGIN_ADJUSTMENT_FOR_KEYBOARD` flag, WebViewObject will adjust bottom margin.
But if developer using native status bar(for example, using `ApplicationChrome` library), calculated `keyboardHeight` value will smaller than expected.
As a result, bottom margin is always bigger than expected, even if keyboard not opened.

Before image, sorry about screenshot not captured with Bottom navigation bar.

---

![image](https://github.com/gree/unity-webview/assets/20940566/2b5e0171-a587-48ec-adbc-8381bc343f12)
Current version, when using `height`, keyboard not opened.

![image](https://github.com/gree/unity-webview/assets/20940566/8d621c29-f267-40b4-b70d-c00149ab4eac)
Current version, when using `height`, keyboard opened.

As you can see, there's margin.

After change `height` to `bottom`:
```c#
// from
            view.Call("getDrawingRect", rect);
            int h0 = rect.Call<int>("height");
            view.Call("getWindowVisibleDisplayFrame", rect);
            int h1 = rect.Call<int>("height");
            keyboardHeight = h0 - h1;

// to
            view.Call("getDrawingRect", rect);
            int h0 = rect.Get<int>("bottom");
            view.Call("getWindowVisibleDisplayFrame", rect);
            int h1 = rect.Get<int>("bottom");
            keyboardHeight = h0 - h1;
```

![image](https://github.com/gree/unity-webview/assets/20940566/aca63ddb-c7e6-4b1c-8cc8-b8054123168f)
After change, when using `bottom`, keyboard opened (previous one on back).

Not checked/tested with `relative` flag.

**Comment by KojiNakamaru** [comment]
getDrawingRect() returns a rect whose (x, y) is usually (0, 0), relative to the parent view/layout. getWindowVisibleDisplayFrame() on the other hand returns a rect whose (x, y) is relative to the screen. As these rects don't have the same origin, using their bottoms here seems wrong.

I might miss something. Based on the sample app, could you please provide a project (including "Resolution and Presentation" settings) that reproduces the issue? You can attach a zip archive to this issue.

**Comment by WolfgangKurz** [comment]
[with_applicationchrome_sample.zip](https://github.com/user-attachments/files/15863070/with_applicationchrome_sample.zip)

Here's problem sample project.
`Windowed`, No `Hide Navigation Bar`, `Portrait` Orientation, with `UNITYWEBVIEW_ANDROID_FORCE_MARGIN_ADJUSTMENT_FOR_KEYBOARD` symbol.

Top margin adjustment solution are in code, bottom margin are not.

Changing `height` to `bottom` is just my one of solution.
Seems each margins calculated separately when set, but getting screen `height` for calculate keyboard size is related with `top` and `bottom`.
That is not problem when fullscreen, but in my case, `top` is not zero anymore (showing status bar with ApplicationChrome.statusBarState), seems that is cause of bottom margin problem.

:)

**Comment by KojiNakamaru** [comment]
Thank you for providing an archive and your detailed explanation. I was able to reproduce the issue, fixed it in #1077, and updated binaries. Could you please try the latest.

**Comment by WolfgangKurz** [comment]
Latest one works very well!
Thank you for your quick work 👍
